where_is_my_sddm_theme/Main.qml

```diff
@@ -25,6 +25,7 @@ Rectangle {
     property string helpFontSize: config.intValue("helpFontSize") || 18
     property string defaultFont: config.stringValue("font") || "monospace"
     property string helpFont: config.stringValue("helpFont") || defaultFont
+    property string helpVisible: config.stringValue("showHelp") || defaultFont
```

Comment:
Wrong name `defaultFont` in this line. Default value should be `false` by default

Reply:
Also should change `stringValue` to `boolValue`